cinema/views.py

```diff
@@ -167,6 +185,23 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "movie",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by movie id (ex. ?movie=1)"
+            ),
+            OpenApiParameter(
+                "date",
+                type={"type": "list", "items": {"type": "date"}},
+                description="Filter movie session by date (ex. ?date=%Y-%m-%d)"
```

Comment:
Movie and date are str parameters, not list

Reply:
Done!